keyboards/levinson/keymaps/jyh/keymap.c

```diff
@@ -0,0 +1,232 @@
+#include "levinson.h"
+#include "action_layer.h"
+#include "eeconfig.h"
+
+
+extern keymap_config_t keymap_config;
+
+// Each layer gets a name for readability, which is then used in the keymap matrix below.
+// The underscores don't mean anything - you can have a layer called STUFF or any other name.
+// Layer names don't all need to be of the same length, obviously, and you can also skip them
+// entirely and just use numbers.
+#define _QWERTY 0
+#define _LOWER 3																/* Symbols, Media */
+#define _RAISE 4																/* Numbers, Arrows */
+#define _FUNC  5																/* Function Keys */
+#define _NUMS  6																/* Numpad */
+#define _ADJUST 16
+
+enum custom_keycodes {
+  QWERTY = SAFE_RANGE,
+  LOWER,
+  RAISE,
+  ADJUST,
+};
+
+// Fillers to make layering more clear
+#define KC_ KC_TRNS
+#define _______ KC_TRNS
```

Comment:
QMK defines these keycode aliases automatically. Lines 28 and 29 can be deleted.

https://github.com/qmk/qmk_firmware/blob/a0309db983150fd0197eb620a6ba552d90d29f93/quantum/quantum_keycodes.h#L29-L31